atv/tests/factories.py

```diff
@@ -3,6 +3,8 @@
 
 
 class GroupFactory(factory.django.DjangoModelFactory):
+    name = factory.Sequence(lambda n: "Group %d" % n)
```

Comment:
The reason to keep this without an implementation was that, since we rely on the `Group` name to be the same as the `Service` name, so it would always "force" to pass a name which then would ensure it's the same as the service name.

Reply:
Indeed this can be removed now. I added before I noticed there was the requirement for the group name to be the same with the service.